tests/e2e/volumepropverification_test.go

```diff
@@ -71,7 +71,7 @@ func TestVolumeProperties(t *testing.T) {
 		log.Println("running end_to_end tests - current time: ", time.Now())
 		vmIP = vms[vmIndx]
 		log.Println("Running test on VM - ", vmIP)
-		dockerCliCheck = verification.IsDockerCliCheckNeeded(vms[vmIndx])
+		dockerCliCheck = true
```

Comment:
this is to remove `IsDockerCliCheckNeeded ` quickly; it will be completely removed as part of #1096 ... as pointed out  https://github.com/vmware/docker-volume-vsphere/pull/1300#issue-231476944